Formula/open-mpi.rb

```diff
@@ -5,6 +5,31 @@ class OpenMpi < Formula
   sha256 "663450d1ee7838b03644507e8a76edfb1fba23e601e9e0b5b2a738e54acd785d"
   revision 1
 
+  stable do
+    # Fix https://github.com/Homebrew/homebrew-core/issues/26009 and its upstream counterpart
+    # https://github.com/open-mpi/ompi/issues/5000 for the time being.  
+
+    # (Remove this patch when either Open MPI v3.0.2 or v3.1.0 is released, as it will be included 
+    # in both of those releases.)
+    #
+    # From https://github.com/open-mpi/ompi/pull/5001 as backported to Open MPI v3.0.x by 
+    # https://github.com/open-mpi/ompi/pull/5029:
+    patch do
+      url "https://patch-diff.githubusercontent.com/raw/open-mpi/ompi/pull/5029.diff"
+      sha256 "4bcd0f0a71c2bbe7ffb07e5394b5148353235b880e27a49ff1032da0a28db826"
+    end
+
+    # (Remove this patch when either Open MPI v3.0.2 or v3.1._1_ is released, as it will be included 
+    # in both of those releases — but _not_ v3.1._0_!)
+    #
+    # From https://github.com/open-mpi/ompi/pull/5015 as backported to Open MPI v3.0.x by 
+    # https://github.com/open-mpi/ompi/pull/5119:
```

Comment:
&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Similarly, is this comment too long, and how would you prefer it be worded?